### ISSUE TITLE
Consider adding tool: Newick display to support users running GTN 16S tutorial on main

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/usegalaxy-playbook#72]
MTS: https://toolshed.g2.bx.psu.edu/view/dcorreia/newick_display/3200899efad9

Tool: Newick display - Display a phylogenetic tree as SVG v1.6

Tutorial: http://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html#otu-based-analysis

Section: "Tree", the tool Newick display is used after Mothur "Tree.shared Generate a newick tree for dissimilarity among groups (Galaxy Version 1.36.1.0)"

This tool missing from Main is one of the warnings given when the workflow for the tutorial is imported at Main. (source: https://github.com/galaxyproject/training-material/tree/master/topics/metagenomics/tutorials/mothur-miseq-sop/workflows/)

@bgruening Is there a reason why this tool is also not included at https://galaxy.uni-freiburg.de or why we should avoid using it at https://usegalaxy.org? 

@nekrut @jxtx What do you think about adding this tool?

**Comment by dfornika** [comment]
I had some problems installing [newick_display](https://toolshed.g2.bx.psu.edu/repository?repository_id=9c5c45b2d41dd665&changeset_revision=3200899efad9) because it lists both `newick_utils` and `newick_utilities` as dependencies, but only `newick_utils` is available on conda. I've created an alternative repo that only depends on `newick_utils` [here](https://toolshed.g2.bx.psu.edu/view/dfornika/newick_display/2073ce44237f).

**Comment by jennaj** [comment]
I don't see either version on main yet. @davebx is this in progress? Wasn't sure about the decision to add or not.

**Comment by natefoo** [comment]
A tool/IUC person should probably review this prior to installation.

**Comment by martenson** [comment]
xref: https://github.com/galaxyproject/galaxy/pull/6206 ?

**Comment by jennaj** [comment]
Good point. I just checked `eu` and they don't have it installed, either. @erasche or @bgruening Do you know why?

Does the tool need a fix? Should the IUC review @dfornika's version and see if can incorporate? Or we just use it directly? Could put it on the Test server and try it out?

**Comment by dfornika** [comment]
I've uploaded my version to github [here](https://github.com/dfornika/newick_display). It is based off of [this repo](https://toolshed.g2.bx.psu.edu/repository?repository_id=9c5c45b2d41dd665&changeset_revision=3200899efad9) created by Galaxy Toolshed user 'dcorreia'

**Comment by erasche** [comment]
@shiltemann has incorporated this in IUC. EU will install this weekend.

**Comment by jennaj** [comment]
Hi - just following up. I didn't find the tool at EU yet. Still being worked on? Wanted to help try it out/test there then see if we want to add it to ORG.

**Comment by erasche** [comment]
Hi @jennaj it's [here](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/newick_utils/newick_display/1.6), under Graph/Display (the first one.)

**Comment by jennaj** [comment]
Missed the ping and just found this! I'll check it out at EU and follow up! Thanks all!
